app/models/solidus/affirm/checkout_payload.rb

```diff
@@ -1,3 +1,5 @@
+require 'active_model_serializers'
```

Comment:
Why do we need to do this? This should be auto required by bundler?

Commit message missing so I don't know why this is required here.